pkg/config/v2/upstream.go

```diff
@@ -87,6 +94,11 @@ type Cluster struct {
 	Hosts                []Host              `json:"hosts,omitempty"`
 	ConnectTimeout       *api.DurationConfig `json:"connect_timeout,omitempty"`
 	LbConfig             IsCluster_LbConfig  `json:"lbconfig,omitempty"`
+	DnsRefreshRate       *api.DurationConfig `json:"dns_refresh_rate,omitempty"`
+	RespectDnsTTL        bool                `json:"respect_dns_ttl,omitempty"`
+	DnsLookupFamily      DnsLookupFamily     `json:"dns_lookup_family,omitempty"`
+	DnsResolverFile      string              `json:"dns_resolver_file,omitempty"`
```

Comment:
Could you add a [dns_resolvers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto.html?highlight=strict%20dns) field. If the value of `dns_resolvers` field is empty, the dns resolver address specified by ` DnsResolverFile` defaultly.

Reply:
done